scripts/prepare_gas_network.py

```diff
@@ -930,9 +930,8 @@ def plot_clustered_gas_network(pipelines, bus_regions_onshore):
 
     else:
         print(
-            "Countries:"
-            + bus_regions_onshore.country.unique().tolist()
-            + "has no existing Natral Gas network between the chosen bus regions"
+            "Countries: ".join(bus_regions_onshore.country.unique().tolist())
+            + " has no existing Natral Gas network between the chosen bus regions"
```

Comment:
Maybe
```suggestion
            "The following countries have no existing Natural Gas network between the chosen bus regions:\n"
            + ", ".join(bus_regions_onshore.country.unique().tolist())
```

? The .join has weird effects when used on "Countries: "